googletest/include/gtest/internal/gtest-internal.h

```diff
@@ -307,7 +307,7 @@ class FloatingPoint {
   // around may change its bits, although the new value is guaranteed
   // to be also a NAN.  Therefore, don't expect this constructor to
   // preserve the bits in x when x is a NAN.
-  explicit FloatingPoint(const RawType& x) { u_.value_ = x; }
+  explicit FloatingPoint(const RawType& x) : u_() { u_.value_ = x; }
```

Comment:
What does it mean to default-construct a union of RawType and Bits?
It's going to create a default-valued RawType, then overwrite it with 'x'.
I am not sure of the usefulness, portability, or legality of 'u_()'.
The eff-c++ flag might be too picky here? What does it say?